tests/e2e/json_rpc_test.go

```diff
@@ -303,6 +307,26 @@ func (s *jsonRPCTestSuite) TestSendPrivTransaction() {
 		assert.Equal(s.T(), strings.ToLower(tx.To().String()), toAddr)
 	})
 
+	s.Run("should call eth_sendTransaction, for private Quorum Tx, with a privateFrom alias successfully", func() {
```

Comment:
why do you add a test instead of just modifying your existing test to add the privateFrom field?